Box.V2/Models/BoxFolder.cs

```diff
@@ -113,5 +115,17 @@ public class BoxFolder : BoxItem
         /// </summary>
         [JsonProperty(PropertyName = FieldIsExternallyOwned)]
         public Boolean? IsExternallyOwned { get; set; }
+
+        /// <summary>
+        /// The date when the folder will be automatically deleted due to item expiration settings.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldExpiresAt)]
+        public DateTime? ExpiresAt { get; protected set; }
+
+        /// <summary>
+        /// The date when the folder will be automatically deleted due to item expiration settings.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldIsCollaborationRestrictedToEnterprise)]
+        public bool? IsCollaborationRestrictedToEnterprise { get; protected set; }
```

Comment:
Should this be `Boolean?`

Reply:
In C#, `bool` is just an alias for `Boolean` — `bool?` is correct and allows a `null` value.